src/idsk/components/customer-surveys/customer-surveys.js

```diff
@@ -0,0 +1,212 @@
+    this.$module = $module;
+}
+
+CustomerSurveys.prototype.init = function () {
+    var $module = this.$module;
+    var $nextButton = $module.querySelector('#idsk-customer-surveys__send-button');
+    var $previousButton = $module.querySelector('#idsk-customer-surveys__previous-button');
+    var $textAreaFirst = $module.querySelector('.idsk-customer-surveys-text-area #first');
+    var $textAreaSecond = $module.querySelector('.idsk-customer-surveys-text-area #second');
+    var $textAreaThird = $module.querySelector('.idsk-customer-surveys-text-area #third');
+    var $textAreaFourth = $module.querySelector('.idsk-customer-surveys-text-area #fourth');
+    var $radioButtonWork = $module.querySelector('.idsk-customer-survey__radio--work');
+    var $radioButtonPrivate = $module.querySelector('.idsk-customer-survey__radio--private');
+    var $counter = 7;
+    $module.sendButtonDisabled = new Array(7);
+    $module.textAreaMap = new Map()
+
+    if (!$module) {
+        return;
+    }
+
+    for (var index = 0; index < $module.sendButtonDisabled.length; index++) {
+        $module.sendButtonDisabled[index] = false;
+    }
+
+    this.handleCounterOfSubtitles.call(this, $counter);
+
+    if ($radioButtonWork) {
+        var $self = this;
+        // Handle $radioButton click events
+        $radioButtonWork.addEventListener('click', $self.handleRadioButtonWorkClick.bind($self));
+    }
+
+    if ($radioButtonPrivate) {
+        var $self = this;
+        // Handle $radioButton click events
+        $radioButtonPrivate.addEventListener('click', $self.handleRadioButtonPrivateClick.bind($self));
+    }
+
+    if ($nextButton) {
+        $nextButton.addEventListener('click', this.handleNextButtonClick.bind(this));
+    }
+
+    if ($previousButton) {
+        $previousButton.addEventListener('click', this.handlePreviousButtonClick.bind(this));
+    }
+
+    if ($textAreaFirst) {
+        $module.textAreaMap.set('first', 1);
+        $textAreaFirst.addEventListener('input', this.handleStatusOfCharacterCountButton.bind(this, 'first'));
+    }
+
+    if ($textAreaSecond) {
+        $module.textAreaMap.set('second', 2);
+        $textAreaSecond.addEventListener('input', this.handleStatusOfCharacterCountButton.bind(this, 'second'));
+    }
+
+    if ($textAreaThird) {
+        $module.textAreaMap.set('third', 4);
+        $textAreaThird.addEventListener('input', this.handleStatusOfCharacterCountButton.bind(this, 'third'));
+    }
+
+    if ($textAreaFourth) {
+        $module.textAreaMap.set('fourth', 5);
+        $textAreaFourth.addEventListener('input', this.handleStatusOfCharacterCountButton.bind(this, 'fourth'));
+    }
+};
+
+CustomerSurveys.prototype.handleStatusOfCharacterCountButton = function (e, $name) {
+    var $module = this.$module;
+    var $textAreaCharacterCount = $module.querySelector('#' + $name.srcElement.id);
+    var $remainingCharacterCountMessage = $module.querySelector('#' + $name.srcElement.id + '-info');
```

Comment:
pokial spravne chapem, tak ty posielas ako $name parameter len e.g "first" etc.. naco potom pozadujes srcElement.id?
nechcel si len ze hodnota v querySelectore mat vo vysledku '#fisrt'? pototm to staci len takto $module.querySelector('#' + $name);

Reply:
toto tu nic?